pkg/users/logout-account-dialog.js

```diff
@@ -35,7 +35,7 @@ export const logoutAccountDialog = (account) => {
                 style: "primary",
                 caption: _("Log out"),
                 clicked: () => {
-                    return cockpit.spawn(["/usr/bin/loginctl", "terminate-user", account.name], { superuser: "try", err: "message" })
+                    return cockpit.spawn(["loginctl", "terminate-user", account.name], { superuser: "try", err: "message" })
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18429-20230301-083127-e6f41828-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)